test/server/cards/01_SOR/units/FirstLegionSnowTrooper.spec.js

```diff
@@ -0,0 +1,73 @@
+describe('First Legion Snow Trooper', function() {
+    integration(function() {
+        describe('First Legion Snow Troopers ability', function() {
```

Comment:
nit, grammar: should be `'First Legion Snow Trooper\'s ability'`

Reply:
fixed the text